### ISSUE TITLE
Typing simulations

### ISSUE THREAD

**Issue by Bedvin1** [val92130/Supreme-Auto-Checkout#138]

**Comment by Bedvin1** [comment]
I always auto-fill adress and only type cc and cvv and it works. Adress and everything take up too much time to type, because some latency is needed between symbols.

**Comment by Bedvin1** [comment]
Also I cant find an on/off-button, is there one? How may I know its on?

**Comment by xxxsomethingxd** [comment]
Simulated typing is always on and can't be turned off, since latest supreme anti-bot features actually detected the previous typing mechanics of the bot

**Comment by Bedvin1** [comment]
And what about "checkout delay"? Do I have to worry bot will try to checkout before he typed everything?

**Comment by xxxsomethingxd** [comment]
No, bot will first fill the page, then checkout delay timer will start.

**Comment by Bedvin1** [comment]
So its better to make it more short? I dont think 3500ms is still needed after it typed everything. How much time approximately it takes to type everything?

**Comment by mapooo** [comment]
Unless the item is super hyped (eg. Upcoming bogo sweater) having a 3.5s checkout delay likely won't hurt, as most sellout times are about 30 seconds upwards, and typically you would prefer the reliability of the bot over speed where appropriate. Alternatively you could create multiple chrome users or autocop tasks with different delays if you want to ensure some success. Tldr, adjust delay to drop conditions

**Comment by Bedvin1** [comment]
Had some test today. Solved some recaptchas before drop, then left 1.5s delay for checkout and tried auto-cop - no captcha. So its not because of delay. But maybe i would got "violation of rules", idk. Think i`ll just leave it at 3s, since bot got 1 item from open tab to buy in about 8 seconds, which is great.

**Comment by mapooo** [comment]
It can vary based on the item as well. For example bogos are much more likely to need captcha or will ToS pretty easily, so I'd stay within the 3+ second range unless you have another instance running as a backup
